### PR TITLE
Nf 391 eis payloads

### PR DESCRIPTION
**Pull request by njmoorewitchcat**

### CODE REVIEW

File: app/uk/gov/hmrc/nationalimportdutyadjustmentcentre/models/eis/EISCreateCaseResponse.scala [hunk 1]
```diff
@@ -27,25 +27,18 @@ case class EISCreateCaseSuccess(CaseID: String, ProcessingDate: Instant, Status:
...
-  CorrelationID: Option[String] = None,
-  ProcessingDate: Option[Instant] = None
-) extends EISCreateCaseResponse
+case class EISCreateCaseError(errorDetail: EISErrorDetail) extends EISCreateCaseResponse
 
 object EISCreateCaseError {
 
   def fromStatusAndMessage(status: Int, message: String): EISCreateCaseError =
-    EISCreateCaseError(status.toString, message)
+    EISCreateCaseError(EISErrorDetail(Some(s"STATUS${status.toString}"), Some(message)))
```
Comment (njmoorewitchcat) [inline app/uk/gov/hmrc/nationalimportdutyadjustmentcentre/models/eis/EISCreateCaseResponse.scala:39]: OBSERVATION: As this errorCode is not a status code, I added STATUS to the front of any status issues that we generate to make it clear these are errorCodes to be consumed at the front.  If this is not good, let me know and will revert it back to stringing the int.  Only happens we our code has an error in parsing the JSON or get an object back we do not understand.

Comment (SteveSugden) [reply app/uk/gov/hmrc/nationalimportdutyadjustmentcentre/models/eis/EISCreateCaseResponse.scala:39]: Seems reasonable to me.  I take it the FE is not responding to these codes at the moment anyway?
Tip - you don't need `toString` on the Int if you are using it as part of constructing a string.

File: app/uk/gov/hmrc/nationalimportdutyadjustmentcentre/models/eis/EISUpdateCaseResponse.scala [hunk 2]
```diff
@@ -32,17 +32,12 @@ object EISUpdateCaseSuccess {
...
-  CorrelationID: Option[String] = None,
-  ProcessingDate: Option[Instant] = None
-) extends EISUpdateCaseResponse
+case class EISUpdateCaseError(errorDetail: EISErrorDetail) extends EISUpdateCaseResponse
 
 object EISUpdateCaseError {
 
   def fromStatusAndMessage(status: Int, message: String): EISUpdateCaseError =
-    EISUpdateCaseError(status.toString, message)
+    EISUpdateCaseError(EISErrorDetail(Some(s"STATUS${status.toString}"), Some(message)))
```
Comment (SteveSugden) [inline app/uk/gov/hmrc/nationalimportdutyadjustmentcentre/models/eis/EISUpdateCaseResponse.scala:40]: Ditto

**Review by SteveSugden — APPROVED** [review]
LGTM (albeit I'm not 100% familiar with the new shape of the EIS error message)

**Pull request merged**